pkg/workloadmeta/types.go

```diff
@@ -86,9 +90,10 @@ type Kind string
 
 // Defined Kinds
 const (
-	KindContainer     Kind = "container"
-	KindKubernetesPod Kind = "kubernetes_pod"
-	KindECSTask       Kind = "ecs_task"
+	KindContainer              Kind = "container"
+	KindKubernetesPod          Kind = "kubernetes_pod"
+	KindECSTask                Kind = "ecs_task"
+	KindContainerImageMetadata Kind = "container_image_metadata"
```

Comment:
maybe we can name the kind `container_image` 
because we are only in workloadmetadata, so "metadata" is a bit redundant.
WDYT?

Reply:
The `ContainerImage` struct already exists. It's the one that we use to attach basic image info to containers:
https://github.com/DataDog/datadog-agent/blob/e943e522e596edb487c9488399723ea0e89342ca/pkg/workloadmeta/types.go#L246